lib/qtermwidget.json

```diff
@@ -0,0 +1,8 @@
+{
+   "Name" : "QTermWidget",
+   "Version" : "1.0.0",
```

Comment:
Not directly related to this PR but for the release manager @tsujan: another version string besides the one in PyQt bindings! Maybe it's time to write some documents about steps needed for each release.

Reply:
Good to know! There are some things that can be done at CI level to facilitate having only one version in one place. Maybe worth to check it in another PR.

Reply:
> another version string besides the one in PyQt bindings!

I'll ask you to take a look at the prerelease PR when it's ready (soon).